src/Microsoft.Framework.ApplicationHost2/Program.cs

```diff
@@ -145,6 +148,7 @@ private bool ParseArgs(string[] args, out ApplicationHostOptions options, out st
         {
             var app = new CommandLineApplication(throwOnUnexpectedArg: false);
             app.Name = typeof(Program).Namespace;
+            var optionDebug = app.Option("--debug", "Wait for a debugger to attach before executing the application", CommandOptionType.NoValue);
```

Comment:
Ok, I'll kill it here and open a new PR to add to the bootstrapper. I don't want changes that will affect old runtime in with this PR :).